charts/trino/values.yaml

```diff
@@ -727,6 +727,53 @@ kafka:
   #    }
   # ```
 
+jmx:
+  # -- Set to true to enable the RMI server to expose Trino's [JMX metrics](https://trino.io/docs/current/admin/jmx.html).
+  enabled: false
+  registryPort: 9080
+  serverPort: 9081
+  exporter:
+    # jmx.exporter.enabled -- Set to true to export JMX Metrics via HTTP for [Prometheus](https://github.com/prometheus/jmx_exporter) consumption
+    enabled: false
+    image: bitnami/jmx-exporter:latest
+    pullPolicy: Always
+    port: 5556
+    configProperties: []
+    # jmx.exporter.configProperties -- JMX Config Properties is mounted to /etc/jmx-exporter/jmx-exporter-config.yaml
+    # @raw
+    # Example:
+    # ```yaml
+    #  configProperties: |-
+    #     startDelaySeconds: 0
+    #     ssl: false
+    #     lowercaseOutputName: false
+    #     lowercaseOutputLabelNames: false
+    #     includeObjectNames: ["java.lang:type=Threading"]
+    #     autoExcludeObjectNameAttributes: true
+    #     excludeObjectNameAttributes:
+    #       "java.lang:type=OperatingSystem":
+    #         - "ObjectName"
+    #       "java.lang:type=Runtime":
+    #         - "ClassPath"
+    #         - "SystemProperties"
+    #     rules:
+    #       - pattern: 'java\.lang<type=Threading><(.*)>ThreadCount: (.*)'
+    #         name: java_lang_Threading_ThreadCount
+    #         value: '$2'
+    #         help: 'ThreadCount (java.lang<type=Threading><>ThreadCount)'
+    #         type: UNTYPED
+
+serviceMonitor:
+  # serviceMonitor.enabled -- Set to true to create resources for the [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator).
+  enabled: false
+  # serviceMonitor.selector -- Labels for serviceMonitor, so that Prometheus can select it
+  selector:
+    prometheus: kube-prometheus
+  # serviceMonitor.path -- The serviceMonitor web endpoint path
+  path: /admin/metrics
```

Comment:
Is this the JMX Exporter path? If yes, how this is configurable in the JMX Exporter? Doesn't it default to `/metrics`? I can't find any references in the Prometheus Operator docs.

Reply:
Yes, you're correct. It's on the [CRD](https://github.com/prometheus-community/helm-charts/blob/b651995181664fef3be46ef52ab963fb7899a009/charts/kube-prometheus-stack/charts/crds/crds/crd-servicemonitors.yaml#L455-L460)

Reply:
We only need to support what we test, so let's not worry about OpenMetrics for now.